processors/games/Bedwars.js

```diff
@@ -68,6 +68,7 @@ module.exports = ({
     coins,
     exp: Experience,
     level: getLevelForExp(Experience),
+    level_formatted: getLevelFormatted(getLevelForExp(Experience)),
```

Comment:
Just a minor optimization thing - it would make sense to make a temporary variable to store level so it doesn't get calculated twice

Reply:
Fixed :)